pages/order.tsx

```diff
@@ -316,20 +312,17 @@ const getTransactionsToDisplay = (account, transactions) => {
   return [...accountTransactions, ...tipTransactionsToDisplay];
 };
 
-export default function OrderPage(props: OrderPageQuery & { error: any }) {
+export default function OrderPage(props) {
```

Comment:
I'm still getting `Error: Hydration failed because the initial UI does not match what was rendered on the server.` on this page. For both when I'm logged in and logged out.

I think this is OK if the page doesn't break in production. But we'll need to eventually tweak to remove this.

Reply:
It's likely related to https://github.com/opencollective/opencollective/issues/7003, but I'll double check

Reply:
I double-checked, and the error is there already on staging: https://staging.opencollective.com/apex/contributions/51029. It's the intl/node issue reported above.